components/automate-ui/src/app/pages/+compliance/+scanner/containers/nodes-add/nodes-add.component.ts

```diff
@@ -66,11 +66,12 @@ export class NodesAddComponent implements OnInit {
     });
 
     // Switch selectable secrets based on selected "backend" value (ssh, winrm)
-    this.secrets$ = combineLatest(
+    this.secrets$ = combineLatest([
       this.fetchSecrets(),
       this.backendValue
-    ).pipe(
-      map(([secrets, backend]) => secrets.filter(s => s.type === backend || s.type === 'sudo'))
+    ]).pipe(
+      map(([secrets, backend]: [any[], string]) =>
```

Comment:
how about
```suggestion
      map(([secrets, backend]: [{ type: string }[], string]) =>
```

seems to be an OK thing to do according to the playground: https://www.typescriptlang.org/play/index.html#code/MYewdgzgLgBBCmwBO8oQFwwN4ygTwAd5NokBLMAcxgF8BtAXRgF4Y6AoGL7XQ4mAEQQIACwG12DIA